src/features/zoom_to/index.tsx

```
@@ -0,0 +1,62 @@
+  },
+});
+
+function geometryExists(geometry: GeoJSON.GeoJSON | null | undefined) {
+  return !isGeoJSONEmpty(geometry);
+}
+
+zoomToControl.onStateChange((ctx, state) => {
+  if (state === 'active') {
+    const geometry = store.v3ctx.get(focusedGeometryAtom)?.geometry;
+    if (geometryExists(geometry)) {
+      const bbox = getBboxForGeometry(geometry!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)